rules/windows/process_creation/proc_creation_win_msdt_arbitrary_command_execution.yml

```diff
@@ -1,4 +1,4 @@
-title: Potential Arbitrary Command Execution Using MSDT.EXE
+title: Potential Arbitrary Command Execution Using Msdt.EXE
```

Comment:
modified too.
Not a big change but if the tools is case sensitive.

Reply:
No need for modified here only casing changed

Reply:
A title change shouldn't even be an issue in the first place if we're really being pedantic let a lone casing change

In reality, only the logic, UUID, and level should impact the `modified` because they are the defining aspect of a rule. A title change similar to a description and references is just metadata ;) 

But I will change it nonetheless

Reply:
We didn't have UUIDs from day one. So there are (or might be) some downstream tools still using the title as identifier. Maybe we can formalize that in the next iteration of the spec. But as we don't do versioning right now, we have to take care of the restrictions due to legacy stuff.

Reply:
In principle I agree with you.